beacon-chain/sync/validate_blob.go

```diff
@@ -92,8 +92,11 @@ func (s *Service) validateBlob(ctx context.Context, pid peer.ID, msg *pubsub.Mes
 		return pubsub.ValidationIgnore, err
 	}
 
-	if err := vf.ValidProposerSignature(ctx); err != nil {
-		return pubsub.ValidationReject, err
+	// TODO: verify the proposer signature here by fetching it from the chunks cache, or putting the blob in a queue
```

Comment:
We need to update the logic to put in a pending queue when no chunk has been seen, and if a chunk has been seen, we just need to check that the signature we keep in the node is the same as in the sidecar.

Reply:
Ultimately the right solution is to change the `BeaconBlockHeader` structure to be the one in the chunk, and also change the layout of the `BeaconBlockChunkHeader` to keep only the commitments roots and not the full commitments.